README.adoc

```diff
@@ -551,6 +551,31 @@ service.expects(:execute).returns(true)
 
 Choose only one to use – avoid mixing both approaches within one project.
 
+== Subclasses might run the parent tests methods twice.
```

Comment:
I think the section header should only mention the topic area and not be too specific, e.g.:
```suggestion
== Subclassing Test Cases
```